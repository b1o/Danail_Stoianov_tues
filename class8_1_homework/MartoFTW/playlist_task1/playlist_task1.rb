require "csv"

out_arr = []

	CSV.foreach(ARGV[0]) do |row|
		if row[2].to_s == ARGV[1].to_s
			out_arr << row
		end
	end

	out_arr.each do |elem|
		elem[3] = elem[3].to_f
	end

	CSV.open("playlist_task1_result.csv","w") do |csv|
		out_arr.sort_by{|a,b,c,d| d}.each do |elem|
			csv << elem
		end	
	end
