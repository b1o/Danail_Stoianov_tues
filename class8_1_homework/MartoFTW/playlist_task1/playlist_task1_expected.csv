Hey you,Pink Floyd,Rock,3.23
Coming back to life,Pink Floyd,Rock,3.31
Comfortably numb,Pink Floyd,Rock,3.45
On the turnin away,Pink Floyd,Rock,4.12
Wind of change,Scorpions,Rock,4.34
